README.md

```diff
@@ -102,7 +101,7 @@ This project will hence provide a CLI tool and a Rust library which speaks the D
 An important aspect of this is that the system will validate the GPG signature of the target OSTree commit, as well as validating the sha256 of the contained objects.
 
 ```
-$ ostree-container pull --repo=/ostree/repo --ref=exampleos/x86_64/stable quay.io/exampleos/exampleos:stable
+$ ostree-ext-cli container import --repo=/ostree/repo quay.io/exampleos/exampleos:stable
```

Comment:
Not super familiar with `ostree-ext-cli`, but is a ref not required here anymore?

Reply:
I wrote the whole README.md first as an aspirational thing, and then wrote the code.  The code doesn't current support `--ref` but it totally should.

Reply:
(Just trying to correct README.md to reflect reality today)